src/main/java/io/stargate/sgv2/jsonapi/util/CqlVectorUtil.java

```diff
@@ -0,0 +1,62 @@
+package io.stargate.sgv2.jsonapi.util;
+
+import com.datastax.oss.driver.api.core.data.CqlVector;
+import java.nio.ByteBuffer;
+import java.util.*;
+
+/**
+ * Static utility methods for working with vector value representations; specifically with Cassandra
+ * {@code float}-based vector types.
+ */
+public interface CqlVectorUtil {
+  static CqlVector<Float> bytesToCqlVector(byte[] packedBytes) {
+    return floatsToCqlVector(bytesToFloats(packedBytes));
+  }
+
+  static float[] bytesToFloats(byte[] packedBytes) throws IllegalArgumentException {
```

Comment:
Can you add some comments on what is done?

Reply:
Will do.